File: server-mvc/server-mvc/Models/Transaction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace server_mvc.Models
{
    public class Transaction
    {
        public Transaction() { }

        public int Id { get; set; }
        public string UserId { get; set; }
        public string TransType { get; set; }
        public string Description { get; set; }
        public decimal Amount { get; set; }
        public decimal Tax { get; set; }
        public string Category { get; set; }
        public DateTime TransDate { get; set; }
    }
}

File: client-vue/src/models/transaction.js
export default class Transaction {
  constructor (id, transType, description, amount, tax, transDate, category, isPersonal = false, isRemove = false, isOutOfPocket = false) {
    this.id = id
    this.transDate = transType
    this.description = description
    this.amount = amount
    this.tax = tax
    this.transDate = transDate
    this.category = category
    this.isPersonal = isPersonal
    this.isRemove = isRemove
    this.isOutOfPocket = isOutOfPocket
  }
}
